classes/rootfs-image-generator.bbclass

```diff
@@ -82,6 +82,9 @@ do_create_rootfs_image() {
         init_ostree_repo
     fi
 
+    ROOTFS_FULL_PATH=$(realpath ${ROOTFS_SOURCE_DIR})
```

Comment:
Does ROOTFS_SOURCE_DIR contain a relative path? And doesn't it work with a relative path?

Reply:
It contains symlink for rootfs and unfortunately, setfiles doesn't work with symlinks.